Pipfile

```diff
@@ -47,7 +48,7 @@ kombu = "==4.2.1"
 linear-tsv = "==1.1.0"
 mo-future = "==2.20.18317"
 moz-sql-parser = "==2.19.18318"
-numpy = "==1.15.2"
+numpy = "==1.17.2"
```

Comment:
This looks good but can we ensure same changes are their in[ setup.py ](https://github.com/Guavus/incubator-superset/blob/fix/MIQ-2130-add-dependency-flask-cors/setup.py#L118)

We build docker using this. can we add numpy update here and test a docker build